pkgimage.mk

```diff
@@ -7,7 +7,7 @@ include $(JULIAHOME)/stdlib/stdlib.mk
 
 # set some influential environment variables
 export JULIA_DEPOT_PATH := $(build_prefix)/share/julia
-export JULIA_LOAD_PATH := @stdlib
+export JULIA_LOAD_PATH := @stdlib:$(JULIAHOME)/stdlib
```

Comment:
Why is this necessary?

Reply:
It doesn't know about the extensions (SparseArraysExt and REPLExt) if this isn't like this. Not sure I 100% understand, but this did fix that issue. @KristofferC sound reasonable?

Reply:
Hm, ideally it should find them even with only `@stdlib`. Not sure why this would fix it.

Reply:
Probably about having the right active project? Note that the Project.toml and Manifest.toml sit in `$(JULIAHOME)/stdlib/`

Reply:
Should we instead set `JULIA_PROJECT`?

Reply:
> Probably about having the right active project? Note that the Project.toml and Manifest.toml sit in $(JULIAHOME)/stdlib/

Okay, in that case it makes sense I think.

Reply:
Would that be different to `--project=$(JULIAHOME)/stdlib` which is what I had before that didn't work